docs/types/mapping-types.rst

```diff
@@ -7,9 +7,8 @@ Mapping Types
 Mapping types use the syntax ``mapping(_KeyType => _ValueType)`` and variables
 of mapping type are declared using the syntax ``mapping(_KeyType => _ValueType) _VariableName``.
 The ``_KeyType`` can be any
-built-in value type plus ``bytes`` and ``string``. User-defined
-or complex types such as contract types, enums, mappings, structs or array types
-apart from ``bytes`` and ``string`` are not allowed.
+built-in value type, ``bytes``, ``string`` or any contract type. Other user-defined
```

Comment:
We actually wanted to include enums.

Reply:
Ok, I can do that...

Reply:
Please change the documentation accordingly.

Reply:
Done now.